### ISSUE TITLE
The GraphConvTensorGraph performs different from cnn_graph model in example

### ISSUE THREAD

**Issue by zealseeker** [deepchem/deepchem#993]
Here is the gist in which I built two models with the same training and test set.
https://gist.github.com/zealseeker/c05b88527cc1ceda7525c2889e8ab47f

The "cnn_graph" model is stable, the performance is reproducible (about 0.69 R2)  while the "tensorgraph" model is worse than the former.

It seems that the architecture of the two models is the same so I wonder how to make the "tensorgraph" as good as the deprecated cnn_graph model (I call it deprecated model because it cannot be saved).

**Comment by rbharath** [comment]
The two versions differ slightly in implementation, which could account for the difference. I'd recommend doing a hyperparameter search on the Tensorgraph model to see if performance improves.

**Comment by zealseeker** [comment]
@rbharath Thanks for the suggestion.
I tried to change the hyper-parameters such as using `dropout=0.2, learning_rate=0.001, dense_layer_size=64, graph_conv_layers=[32, 32]`.
The performance can be better to some extends but all of them were not as good as the previous model. 
Is there any other "important" hyper-parameters that may affect the performance?

**Comment by rbharath** [comment]
I'd recommend a systematic hyperparameter search, along the lines of MoleculeNet param searches.

CC @miaecle who did MoleculeNet hyperparam searches

**Comment by zealseeker** [comment]
Is there any good way to search the hyper-parameters? For example, how to implement `deepchem/deepchem/hyper/grid_search.py`?
Here is the data and script to reproduce the gist. 
[tf_vs_cg.zip](https://github.com/deepchem/deepchem/files/1588260/tf_vs_cg.zip)

**Comment by rbharath** [comment]
I think we've no rerun the TensorGraph graph conv against MoleculeNet and things look stable. Going to close. Feel free to re-open if there are still issues.
